lib/aws-manager.js

```diff
@@ -1300,37 +1151,18 @@ class AwsManager {
     let i = instances || [];
     let r = requests || [];
 
-    if (i.length > 0) {
-      debug('killing instances %s in %j', i, region);
-      try {
-        await this.ec2.terminateInstances.inRegion(region, {
-          InstanceIds: i,
-        });
-      } catch (err) {
-        debug('error terminating instances in %s', region);
-        debug(err);
-        if (err.stack) {
-          debug(err.stack);
-        }
-        throw err;
-      }
-    }
+    let promises = [];
```

Comment:
You don't need this...

Also don't do default values like this... just do `async killCancel (region, instances = [], requests = []) {`